x11-misc/fbpanel/files/fbpanel-7.0-shebangs.patch

```diff
@@ -0,0 +1,135 @@
+diff -u -r a/.config/help b/.config/help
+--- a/.config/help	2015-12-05 08:34:19.000000000 +0200
++++ b/.config/help	2017-02-25 15:18:46.328392853 +0200
+@@ -1,4 +1,4 @@
+-#!/usr/bin/python
++#!/usr/bin/env python2
```

Comment:
:heart: 
Can you try upstreaming this and giving them the rundown why `env` based-lookup is better than hardcoded paths? (https://blogs.gentoo.org/mgorny/2016/02/08/a-quick-note-on-portable-shebangs/)